viewing_party/party.py

```diff
@@ -1,23 +1,175 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
+    
+    movie_dict["title"] = title
+    movie_dict["genre"] = genre
+    movie_dict["rating"] = rating
+
+    return movie_dict
+
+
+def add_to_watched(user_data, movie):
+    watched = user_data["watched"] 
+    if watched is None:        
+        watched = [movie]
+    watched.append(movie)

```

Comment:
The requirements say that the dictionary will have a key "watched" so we can remove the conditional checks here to simplify this function:

```suggestion
    user_data["watched"].append(movie)
```